roles/ovirt-vm-infra/README.md

```diff
@@ -60,27 +68,24 @@ The item in `disks` list of `profile` dictionary can contain following attribute
 | name               | UNDEF          | The name of the additional disk.  |
 | storage_domain     | UNDEF          | The name of storage domain where disk should be created. |
 | interface          | UNDEF          | The interface of the disk. |
+| format             | UNDEF          | Specify format of the disk.  <ul><li>cow - format is used, disk will by created as sparse, so space will be allocated for the volume as needed, also known as thin provision</li><li>raw - format is used, disk storage will be allocated right away, also known as preallocated.</li></ul> |
+| bootable           | UNDEF          | True if the disk should be bootable. |
 
 The item in `nics` list of `profile` dictionary can contain following attributes:
 
-| Name           | Default value  |                                              |
-|----------------|----------------|----------------------------------------------| 
-| name           | UNDEF          | Name of the NIC. |
-| profile_name   | UNDEF          | Profile name where NIC should be attached. |
-| interface      | UNDEF          | Type of the network interface. One of following: virtio, e1000, rtl8139, default is virtio. |
-| mac_address    | UNDEF          | Custom MAC address of the network interface, by default it's obtained from MAC pool. |
-| network        | UNDEF          | Logical network to which the VM network interface should use, by default Empty network is used if network is not specified. |
-
-The item in `cloud_init_nics` list of `profile` dictionary can contain following attributes:
-
 | Name               | Default value  |                                              |
 |--------------------|----------------|----------------------------------------------| 
-| nic_boot_protocol  | UNDEF          | Set boot protocol of the network interface of Virtual Machine. Can be one of none, dhcp or static. |
-| nic_ip_address     | UNDEF          | If boot protocol is static, set this IP address to network interface of Virtual Machine.
-| nic_netmask        | UNDEF          | If boot protocol is static, set this netmask to network interface of Virtual Machine.
-| nic_gateway        | UNDEF          | If boot protocol is static, set this gateway to network interface of Virtual Machine.
-| nic_name           | UNDEF          | Set name to network interface of Virtual Machine.
-| nic_on_boot        | UNDEF          | If True network interface will be set to start on boot.
+| name               | UNDEF          | The name of the network interface.           |
+| interface          | UNDEF          | Type of the network interface.               |
+| mac_address        | UNDEF          | Custom MAC address of the network interface, by default it's obtained from MAC pool. |
+| network            | UNDEF          | Logical network to which the VM network interface should use, by default Empty network is used if network is not specified. |
```

Comment:
Logical network which the VM network interface should use. If network is specified, then Empty network is used.

Reply:
done